app/scrappers.toml

```diff
@@ -0,0 +1,52 @@
+# Scrapper configuration file.
+#
+# You can define new scrappers here by using the pre-defined templates
+# of other imageboards. Parameters are as follows:
+#
+#  The imageboard name Will be used in the database, actor system, logs and other things
+#  and thus must be unique, you can think of it as the scrapper ID
+#   [imageboard-name]
+#
+#  Type is the pre-defined template the scrapper will use
+#   type = "old-danbooru"
+#         "new-danbooru"
+#         "moebooru"
+#
+#  Base URL is the URL all requests by the scrapper will be made to,
+#  it SHOULD NOT include a trailing slash, as this can cause errors
```

Comment:
If it __should__ (not __must__) not, we should add trailing slash removing, and as well get rid of this requirement.
User input tends to be sloppy and  it always least reliable part of the system.